package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Seed project for easily getting started with express for es6",
   "scripts": {
-    "start": "nodemon ./server.js --exec babel-node",
+    "start": "mkdir -p log && nodemon ./server.js --exec babel-node",
```

Comment:
I Think it's problematic in case log files dir already exists... I get the following error on my windows 7 machine:
![erroronprstart](https://user-images.githubusercontent.com/16193919/30830597-2dab6ba6-a24d-11e7-90b8-1d18bc841b20.PNG)

Can you fix it?

Reply:
I see. Yes that is a problem. "mkdir" doesn't behave in all operating systems. Let me see if I can come up with an alternative approach.